famstories/index.html

```diff
@@ -0,0 +1,105 @@
+
+<img
+src="../assets/images/divider.png"
+
+<p></p>
+
+
+<img
+src="../assets/images/choctri2024.jpeg"
+
+<p>Holly P.</p>
+<p>Mar. 2025</p>
+<p>I found Susan through Good dog. I wanted to find a breeder that was careful not to “overbreed” and really treated the puppies like a lover of dogs. Susan was very responsive to all my questions. She could answer all my questions without hesitation and it was clear she knew the Aussie doodle breed. She worked with me on a date for pick up and was flexible when I needed to change that date. She FaceTimed me to allow me to see my puppy’s behavior & personality. I am very happy with Susan & with my sweet little Harper. She is happy, calm, & so loving. Very pleased with my sweet little girl!</p>
+
+
+
+<img
+src="../assets/images/divider.png"
+
+<p></p>
+
+
+<img
+src="../assets/images/Oakley2024.png"
+
+<p>Felicia B.</p>
+<p>Nov. 2024</p>
+
+<p>Our experience with Susan at Doodle Dandy was nothing short of wonderful! From the moment we began the process of bringing Oakley home, Susan was there every step of the way. She kept us informed with constant updates, sent photos, and answered all our questions thoroughly. It’s clear she truly cares about each puppy and each family she works with. Susan even went above and beyond to personally bring Oakley to us in Florida, making sure the transition was smooth. Now that Oakley is here, we’re completely smitten! He has the sweetest, calmest personality but is still so playful and fun. It’s obvious that Susan’s love and care have shaped him into the perfect addition to our family. We’re so grateful to Susan for her dedication, and we’d wholeheartedly recommend Doodle Dandy to anyone searching for their perfect pup!</p>
+<img
+src="../assets/images/divider.png"
+
+<p></p>
+
+
+
+   <h2>
+      <ul>
+        <li>
+          <i class="fa-solid fa-envelope fa-pad-right"></i>
+          <a href="mailto:adopt@doodledandy.dog">adopt@doodledandy.dog</a>
+        </li>
+        <li>
+          <i class="fa-solid fa-phone fa-pad-right"></i>
+          <a href="tel:+14174209882">417-420-9882</a>
+        </li>
+      </ul>
```

Comment:
This is invalid html - at minimum, get rid of the `h2` element.
```suggestion

```

You already have a Contact page - get rid of this whole section.